### ISSUE TITLE
[SCREENREADER]: Text and inputs added dynamically should be announced by screen readers

### ISSUE THREAD

**Issue by peterhill-oddball** [department-of-veterans-affairs/va.gov-team#1072]
## Issue
New inputs or conditional text added to the DOM based on user interaction with radio buttons, checkboxes, or select menus need to be announced to screen readers so users have context that new information is available.

## Steps to Reproduce
1. Review a form with radio buttons or checkboxes that conditionally expose additional content. This might be text or an input field.
2. Verify the new information or fields are not read out by the screen reader

## Related Issues
* https://github.com/usds/us-forms-system/issues/279

## Acceptance Criteria
The new information should be read out by the screen reader. We want users to know the form or view has changed, and additional contextual information is being presented. Screenshots below show a few instances of this pattern.

## Possible Fix
Works in Progress:
* https://github.com/usds/us-forms-system/issues/279#issuecomment-424064503

---

## Screenshots
<img width="546" alt="screen shot 2018-09-20 at 10 26 48 pm" src="https://user-images.githubusercontent.com/934879/45859205-ac2f7b80-bd26-11e8-95c3-a7b758cc91ee.png">

**Comment by peterhill-oddball** [comment]
Migrated from vets.gov issue [316](https://github.com/department-of-veterans-affairs/va.gov-team/issues/316)

**Comment by peterhill-oddball** [comment]
Mistakenly "migrated" when it was already in VA.gov.  Ignore this issue.
